packages/driver/src/cy/commands/screenshot.ts

```diff
@@ -337,7 +337,7 @@ const takeScreenshot = (Cypress, state, screenshotConfig, options: TakeScreensho
     }
   }
 
-  const before = ($el) => {
+  const before = ($body, $container) => {
```

Comment:
Can we incrementally ad types as we go? `HTMLBodyElement` and `HTMLElement`, in this case.

Reply:
Going to create a follow up PR to add types to the screenshot files since it's quite a bit of changes